ipinfo/handler.py

```diff
@@ -46,39 +46,19 @@ def __init__(self, access_token=None, **kwargs):
         self.access_token = access_token
 
         # load countries file
-        self.countries = handler_utils.read_json_file(
-            kwargs.get("countries_file")
-            if kwargs.get("countries_file")
-            else COUNTRY_FILE_DEFAULT
-        )
+        self.countries = countries
 
         # load eu countries file
-        self.eu_countries = handler_utils.read_json_file(
-            kwargs.get("eu_countries_file")
-            if kwargs.get("eu_countries_file")
-            else COUNTRY_EU_FILE_DEFAULT
-        )
+        self.eu_countries = eu_countries
```

Comment:
Let us keep it configurable - allow user to input this themselves too.

Then we must update README.md if it contains info about 'internationalization' to mention that it must be input as a map, not a file path.

Reply:
Please check if the recent solution for this will work. I think with the current structure we would not need to update README.md.